setup.cfg

```diff
@@ -1,26 +1,27 @@
 [metadata]
-description-file = README.md
 name = wagtailextraicons
+version = 3.0.0
 description = Add extra icons to your Wagtail project.
+description-file = README.md
 long_description = file: README.md
 long_description_content_type = text/markdown
+url = https://github.com/octavenz/wagtailextraicons
 author = Sam Costigan (Octave)
 author_email = sam@octave.nz
```

Comment:
Which email should be used here?

Reply:
@Siburg let's change to support@octave.nz please.

Reply:
Done